src/markup/codemirror/create.ts

```diff
@@ -53,6 +53,7 @@ export type CreateCodemirrorParams = {
     uploadHandler?: FileUploadHandler;
     needImgDimms?: boolean;
     extraMarkupExtensions?: Extension[];
+    autocompletionConfig?: Parameters<typeof autocompletion>[0];
```

Comment:
Let's change this prop – we will forward the functions for the autocomplete, not the configuration of the plugin